main/src/ca/uwaterloo/flix/tools/pkg/FlixPackageManager.scala

```diff
@@ -34,7 +35,14 @@ object FlixPackageManager {
     */
   def findTransitiveDependencies(manifest: Manifest, path: Path, apiKey: Option[String])(implicit formatter: Formatter, out: PrintStream): Result[List[Manifest], PackageError] = {
     out.println("Resolving Flix dependencies...")
-    findTransitiveDependenciesRec(manifest, path, List(manifest), apiKey)
+    implicit val immediateDependents: mutable.Map[Manifest, List[Manifest]] = mutable.Map.empty
+    implicit val trustLevels: mutable.Map[Manifest, Trust] = mutable.Map(manifest -> Trust.Unrestricted)
```

Comment:
Do we have to compute this *during* resolution? 

I would have expected we discover the graph and then compute on it.